### DISCUSSION TITLE
Passing tourAnchor to Ag-Grid column

### DISCUSSION THREAD

**Discussion by FlynnDerek** [hakimio/ngx-ui-tour#50] [Q&A] [answered]
I'm attempting to highlight a column in ag-grid. I think in general you should be able to pass a global angular attribute like tourAnchor to the grid's cellRenderer but no luck with this approach.

Has anyone used this library with ag-grid? Would appreciate hearing anyone's thoughts on this, thanks yall.

**Comment by hakimio** [comment, marked as answer]
I imagine you should be able to use [Cell Renderer Component](https://www.ag-grid.com/angular-data-grid/component-cell-renderer/#cell-renderer-component-1) to add Angular directive to a cell, but you can not add the same `tourAnchor` to multiple cells.
Anyway, this is not an issue with this library and if you need help with `ag-grid`, might be a better idea to post a question in their forums.

**Reply by Poulami1998** [reply]
Ok thank you
